packages/typescript-checker/.npmignore

```diff
@@ -0,0 +1,11 @@
+**/*
```

Comment:
what about the dist folder? Wouldn't you want to exclude that?

Reply:
Yep, this is copy-paste error. Thanks